buildrunner/docker/multiplatform_image_builder.py

```diff
@@ -298,6 +305,7 @@ def _build_single_image(
         LOGGER.info(
             f"Building image for platform {platform} with {builder or 'default'} builder"
         )
+        logs_itr = Iterator[str]
```

Comment:
did you mean `logs_itr: Iterator[str]`

Reply:
Or just remove this completely, since it's not used in the other `if` case

Reply:
Yeah it's not needed. I was going to use it in the other if but moved it in `_build_with_inject`. Removed it.